### ISSUE TITLE
YouTube and Plex.tv not showing in text file anymore

### ISSUE THREAD

**Issue by Gamerbiomek2** [Wykks/Untamed-Now-Playing-Next#163]
other websites work for me, but Plex  and YouTube won't for some reason

**Comment by Senpos** [comment]
@Gamerbiomek2 Do you use new YouTube design?
Will take a look at both of them soon.

**Comment by Gamerbiomek2** [comment]
yes I use the new YouTube design

**Comment by Uro1** [comment]
I can confirm Youtube's new theme format breaks this extension, I do like the dark theme though, no more retina burn! 
As a workaround you can via Youtubes' user settings switch back to the old theme meantime which allows the extension to continue to work as normal.

**Comment by Senpos** [comment]
@Wykks i am experiencing some issues fixing this, can you help, please?

Here is required selectors which are actually working in console:

```
YoutubeTrackListener.prototype.isPlaying = function () {
        return $('#movie_player').hasClass('playing-mode');
    };

    YoutubeTrackListener.prototype.scrapPlayData = function () {
        const play = $('#player-container').find('.ytp-title-text').text();
        [this.artistName, this.trackName] = window.UNPCommon.parseArtistTitle(play);
        if (!this.artistName) {
            this.artistName = $('#owner-name').text();
        }
        return true;
    };

    YoutubeTrackListener.prototype.scrapAlbumArt = function () {
        return $('#thumbnail').find('#img').attr('src');
    };

    YoutubeTrackListener.prototype.scrapUrl = function () {
        return $('.ytp-title-link ').attr('href');
    };

    YoutubeTrackListener.prototype.scrapDuration = function () {
        return $('.ytp-time-duration').text();
    };
```

**Comment by Gamerbiomek2** [comment]
I am not good with coding, sorry I can't help

**Comment by Senpos** [comment]
More information about new YouTube design here: https://github.com/Wykks/Untamed-Now-Playing-Next/issues/165#issuecomment-334986253

**Comment by Gamerbiomek2** [comment]
still not putting song info from plex to text files

**Comment by Senpos** [comment]
@Gamerbiomek2 Yaeh, will take a look at Plex today i hope

**Comment by Senpos** [comment]
@Gamerbiomek2 

I have added all the latest fixes, but not merged them yet, so when @Wykks will update extension for the last time - he will be able to check it out. Once official version of extension will release - you will see it here, on GitHub or even on Mozilla's extension page.

For now, if you are interested in using extension **with all the latest fixes**, i can suggest you installing unofficial version of extension, which you can **[download here [ver. with latest fixes on 18.10.2017]](https://drive.google.com/uc?id=0B6HBCntn0EZRWjE0azBYZ0VMLU0)**.

The problem with unofficial (_unsigned_) extension is that you have to install it every time you open your browser. Good thing is all your settings will stay even after restarting a browser. 

If you don't know, how to install unsiged extension to your Firefox -[ here is a short video instruction](https://drive.google.com/uc?id=0B6HBCntn0EZRM3NVSzEyZjZ3SlE).

In your case, both YouTube new layout and Plex (**without artworks**, unfortunately) should work fine.

Please, let me know if this helps.
